FIDO2_make_authenticator_data_rsp.md

```diff
@@ -0,0 +1,24 @@
+## FIDO2 Make Authenticator Data Response
```

Comment:
this file isn't referenced by anything and seems to be obsolete from what I see.

Reply:
Ditto